anyscale_provider/operators/anyscale.py

```diff
@@ -81,6 +81,7 @@ def __init__(
         wait_for_completion: bool = True,
         job_timeout_seconds: float = 3600,
         poll_interval: float = 60,
+        job_queue_config: JobQueueConfig | None = None,
```

Comment:
We should add this in the docstring

Reply:
Done in f2b28465f44ea0688ae1fd158cabe8970cc6fef5